x-pack/plugins/xpack_legacy/common/index.ts

```diff
@@ -4,4 +4,4 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export { xpackInfoRoute } from './xpack_info';
+export * from './eui_styled_components';
```

Comment:
So, a few plugins were still using the `x-pack/legacy/common/eui_styled_components` module. I moved it to the `xpack_legacy` plugin for now. It might make sense to move it to something like `kibana_react` at some point, but ideally this would be done at a later time. cc @elastic/kibana-app-arch